.github/scripts/update-release-notes.js

```diff
@@ -0,0 +1,165 @@
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+var https = require('https');
+const fs = require("fs");
+const execSync = require('child_process').execSync;
+
+const token = process.argv[2];
+
+if(token == undefined) {
+    throw new Error("token is undefined")
+}
+
+function isEarlierThan24Hours(timestampInMilliseconds) {
+    const timestamp = new Date(timestampInMilliseconds);
+    const now = new Date();
+    return timestamp < now - (24 * 60 * 60 * 1000);
+  }
+
+function fetchReleaseNotes(owner,repo, tag, callback) {
+    var optionsget = {
+        host : 'api.github.com', 
+        port : 443,
+        path :  `/repos/${owner}/${repo}/releases/tags/${tag}`, 
+        method : 'GET',
+        headers: {
+            'Accept': 'application/vnd.github+json',
+            'User-Agent': 'server-side',
+            'X-GitHub-Api-Version': '2022-11-28',
+            'Authorization': `Bearer ${token}`,
+          }
+    };
+    var reqGet = https.request(optionsget, function(res) {
+        // console.log("statusCode: ", res.statusCode);
+        var data = [];
+        res.on('data', function(chunk) {
+            data.push(chunk);
+        }).on('end', function() {
+            var buffer = Buffer.concat(data);
+            str = new TextDecoder("utf-8").decode(buffer)
+            responseJson = JSON.parse(str)
+            
+            callback(responseJson.published_at, responseJson.body)
+            
+        });
+     
+    });
+     
+    reqGet.end();
+    reqGet.on('error', function(e) {
+        console.error(e);
+    });
+}
+
+
+function updateBOMReleaseNotes() {
+var optionsget = {
+    host : 'search.maven.org', 
+    port : 443,
+    path :  '/solrsearch/select?q=g:com.adobe.marketing.mobile+AND+a:sdk-bom&core=gav&rows=2&wt=json', 
+    method : 'GET',
+    headers: {
+        'Content-Type': 'application/json',
+        'User-Agent': 'server-side',
+      }
+};
+
+var reqGet = https.request(optionsget, function(res) {
+    // console.log("statusCode: ", res.statusCode);
+ 
+    res.on('data', function(d) {
+        str = new TextDecoder("utf-8").decode(d)
+        responseJson = JSON.parse(str)
+
+        responseJson.response.docs.forEach(element => {
+            if (isEarlierThan24Hours(element.timestamp) == false) {
+                if (element.a == 'sdk-bom') {
+                    const version = element.v
+
+                    fetchReleaseNotes('adobe','aepsdk-commons',`bom-${version}`, (ios1086DateStr, releaseNotes) => {
+                        const date = new Date(ios1086DateStr)
+                        const month = date.getMonth()
+                        const day = date.getDate()
+                        const year = date.getFullYear()
+                        const monthNames = ["Jan", "Feb", "Mar", "Apr", "May", "June", "July", "Aug", "Sept", "Oct", "Nov", "Dec"];
```

Comment:
We have been using the full month name in our release notes. The last two updates have used "Aug" instead of "August" by mistake.